src/main/java/org/skriptlang/skript/bukkit/loottables/elements/expressions/ExprLootTableFromString.java

```diff
@@ -0,0 +1,61 @@
+package org.skriptlang.skript.bukkit.loottables.elements.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.NamespacedKey;
+import org.bukkit.event.Event;
+import org.bukkit.loot.LootTable;
+import org.jetbrains.annotations.Nullable;
+
+public class ExprLootTableFromString extends SimpleExpression<LootTable> {
+
+	static {
+		Skript.registerExpression(ExprLootTableFromString.class, LootTable.class, ExpressionType.PROPERTY,
+			"[the] loot[ ]table[s] %strings%",
+			"%strings%'[s] loot[ ]table[s]"
+			);
```

Comment:
```suggestion
		Skript.registerExpression(ExprLootTableFromString.class, LootTable.class, ExpressionType.COMBINED,
			"[the] loot[ ]table[s] %strings%");
```
Not a huge fan of the second pattern